docs/getting_started.md

```diff
@@ -52,7 +52,9 @@ Also available in Mac, Linux and WSL Homebrew:
 ```bash
 brew install argocd
 ```
-
+!!! note
+    To ensure proper functionality, the Argo CD CLI requires HTTP/2 support. Verify that your CLI is set up to use HTTP/2, especially if connecting to an Argo CD server over HTTPS. Check your system’s configuration or proxy settings if issues arise.
```

Comment:
```suggestion
    To ensure proper functionality, the Argo CD CLI requires HTTP/2 support. Verify that your CLI is set up to use HTTP/2, especially if connecting to an Argo CD server over HTTPS. Check your system’s configuration or proxy settings if issues arise.
```
1. Why do we need to emphasize on usage of http/2 here?
2. Can you add steps that would be required for verification if CLI I set up to use the HTTP/2?
3. Which issue can occur if you just normally install the Argo CD CLI and use it against the Argo CD server over HTTPS?